src/tasks/DockerBuildTaskProvider.ts

```diff
@@ -33,50 +31,21 @@ export class DockerBuildTaskProvider extends DockerTaskProvider {
     // TODO: Skip if image is freshly built
     protected async executeTaskInternal(context: DockerBuildTaskContext, task: DockerBuildTask): Promise<void> {
         const definition = cloneObject(task.definition);
-        definition.dockerBuild = definition.dockerBuild || {};

```

Comment:
Make sure you're invoking the new base getDockerBuildOptions method in any of the child class method overrides to avoid the dockerBuild property being undefined.

Reply:
Or ensure you initialize to a default value if undefined/null.

Reply:
This makes sense, god catch! :sweat:

Reply:
Will do